roles/init_dbserver/tasks/pg_initdb.yml

```diff
@@ -10,6 +10,36 @@
     pg_initdb_options: "{{ pg_initdb_options + ' --waldir=' + pg_wal }}"
   when: pg_wal|length > 0  and not pg_data in pg_wal
 
+- name: Store edb_master_key in a secure location
+  ansible.builtin.copy:
+    content: "{{ edb_master_key }}"
+    dest: "{{ edb_master_key_location_file }}"
```

Comment:
Should this be `"{{ edb_master_key_plain_file }}"`, the variable `edb_master_key_location_file` does not exist

Reply:
Fixed